package/network/config/ltq-vdsl-app/files/dsl_control

```diff
@@ -194,34 +195,39 @@ start_service() {
 
 	eval "xtse=\"\${xtse_xdsl_$annex}\""
 
-	[ -z "${xfer_mode}" ] && xfer_mode=ptm
-
 	case "${xfer_mode}" in
 	atm)
-		LOAD=ltq_atm_vr9
-		UNLOAD=ltq_ptm_vr9
+		tc_layer="1:0x1:0x1_1:0x1:0x1"
```

Comment:
The '-T' argument of dsl_cpe_control defines the startup configuration for TC-Layer related parameters.
Syntax:` -T<TcADSL>:<TcCfgUsADSL>:<TcCfgDsADSL>_<TcVDSL>:<TcCfgUsVDSL>:<TcCfgDsVDSL>`
Variable definitions:
TcADSL, TcVDSL:
1=ATM, 2=PTM/EFM, 4=Auto TC-Layer

TcCfgUsADSL, TcCfgUsVDSL, TcCfgDsADSL, TcCfgDsVDSL: 
1=64/65-octet encapsulation supported
2=64/65-octet encapsulation with pre-emption
3=64/65-octet encapsulation  with short packets

Default: In case of no '-T' option is given, ADSL will be configured in ATM and VDSL in PTM/EFM: `-T1:0x1:0x1_2:0x1:0x1`

So maybe we should drop the '-T' option in default (auto-mode) case.